garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua

```diff
@@ -245,9 +249,13 @@ local function MuteTeam(ply, cmd, args)
 
    local t = tonumber(args[1])
    ply.mute_team = t
-
-   local name = (t != 0) and team.GetName(t) or "None"
-   ply:ChatPrint(name .. " muted.")
+   if t == MUTE_ALL then
+   ply:ChatPrint("All muted.")
+   elseif t == MUTE_NONE or t == TEAM_UNASSIGNED then

```

Comment:
Because if I use `ttt_mute_team TEAM_TEAM_UNASSIGNED` the output would be `Unassigned muted.`.
But in TTT `TEAM_UNASSIGNED` won't be used.